editor/src/components/canvas/canvas-strategies/relative-move-strategy.tsx

```diff
@@ -0,0 +1,191 @@
+import { ElementPath } from 'src/core/shared/project-file-types'
+import { getLayoutProperty } from '../../../core/layout/getLayoutProperty'
+import { MetadataUtils, PropsOrJSXAttributes } from '../../../core/model/element-metadata-utils'
+import { isRight, right } from '../../../core/shared/either'
+import { ElementInstanceMetadata, isJSXElement } from '../../../core/shared/element-template'
+import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
+import { CanvasCommand } from '../commands/commands'
+import { setCssLengthProperty } from '../commands/set-css-length-command'
+import {
+  CanvasStrategy,
+  emptyStrategyApplicationResult,
+  getTargetPathsFromInteractionTarget,
+  strategyApplicationResult,
+} from './canvas-strategy-types'
+import { getDragTargets } from './shared-absolute-move-strategy-helpers'
+
+export const relativeMoveStrategy: CanvasStrategy = {
+  id: 'RELATIVE_MOVE',
+
+  name: () => 'Move (Relative)',
+
+  isApplicable: (canvasState, _interactionState, metadata) => {
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return false
+    }
+
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const meta = MetadataUtils.findElementByElementPath(metadata, last)
+    if (!meta) {
+      return false
+    }
+
+    // should we also support absolute elements, for which we'll do something like `CONVERT_TO_RELATIVE`?
+    return meta.specialSizeMeasurements.position === 'relative'
+  },
+
+  controlsToRender: [],
+
+  fitness: (canvasState, interactionState, _sessionState) => {
+    const { interactionData, activeControl } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag !== null)) {
```

Comment:
For null checks we prefer the `!=` operator to catch `undefined` too (except if you have special reason to only check for null, which you should not have :)
For all other values we use  `!==`

Reply:
https://github.com/concrete-utopia/utopia/pull/2604/commits/ac2a475cfd0a477db8d7720dad35be1d83e47cfa

Reply:
thanks for letting me know about this, will do in the future as well 👍